### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by digawp**
Refactored, too
@nathanajah If there are some parts that is can be removed safely, pls let me know

### CODE REVIEW

File: FeaturesScore.java [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
 
-	public FeaturesScore(Game s) {
-		grids = s.getField();
+		scores[0] = getLandingHeight(s, move);
 
-		//First 10 are colHeight
-		for (int i = 0; i < COLS; i++) {
-			scores[i] = colHeights[i] = checkColHeight(i);
-		}
+		scores[1] = getRowsClearedAfterMove(s);
```
Comment (nathanajah) [inline FeaturesScore.java]: Shouldn't this be getRowsClearedAfterMove(g)?

File: PlayerSkeleton.java [hunk 2]
```diff
@@ -27,14 +30,17 @@ public int pickMove(State s, int[][] legalMoves) {
 			}
 		}
 
+		/*try {
+			Scanner sc = new Scanner(System.in);
+			int tmp = sc.nextInt();
+		}catch(Exception e) {
+		}*/
```
Comment (digawp) [inline PlayerSkeleton.java:37]: @nathanajah Can this be removed? What was this for?

**Comment by jials** [comment]
I think it looks ok.

**Comment by digawp** [comment]
Then meeeeeeeeeeeeeeeeeeeeeeeeeerrrrrrrrrrrrgggggggggggeeeeeeeeeeeeeeeeeeeeeeee so we can start working

**Pull request merged**